src/core_modules/capture-core/components/CardList/CardList.component.js

```diff
@@ -1,18 +1,19 @@
 // @flow
 import React from 'react';
-import type { ComponentType, Element } from 'react';
+import type { ComponentType } from 'react';
 import { withStyles } from '@material-ui/core';
 import { CardListItem } from './CardListItem.component';
 import { makeElementsContainerSelector } from './CardList.selectors';
-import type { CardDataElementsInformation, SearchResultItem } from '../Pages/Search/SearchResults/SearchResults.types';
+import type { CardDataElementsInformation } from '../Pages/Search/SearchResults/SearchResults.types';
```

Comment:
I think it would be better to have this type defined in the CardList "module" itself. Assuming my context here is correct and the CardList is a more generic component (currently used by both the SearchPage, RegistrationPage, Relationship part of the app)

EDIT: Probably out of scope since I can see you only removed `SearchResultItem` part. And also this is a reoccurring problem throughout this "module".

Reply:
Perhaps we create a TECH ticket to clean this up as we did with the GeneralOutput Component?